backend/fms_core/migrations/0041_auto_20220816_1946.py

```diff
@@ -0,0 +1,58 @@
+# Generated by Django 3.1 on 2022-08-16 19:46
+
+from django.conf import settings
+from django.db import migrations, models
+import django.db.models.deletion
+import fms_core.schema_validators
+
+
+class Migration(migrations.Migration):
+
+    dependencies = [
+        migrations.swappable_dependency(settings.AUTH_USER_MODEL),
+        ('fms_core', '0040_v3_10_0'),
```

Comment:
Rename your file to match the naming scheme you might have to update your dependency (0041 may or may not already exist...).